packages/@aws-cdk-testing/cli-integ/lib/with-cdk-app.ts

```diff
@@ -510,7 +511,12 @@ export class TestFixture extends ShellHelper {
     const imageRepositoryNames = stacksToDelete.map(stack => outputFromStack('ImageRepositoryName', stack)).filter(defined);
     await Promise.all(imageRepositoryNames.map(r => this.aws.deleteImageRepository(r)));
 
-    await this.aws.deleteStacks(...stacksToDelete.map(s => s.StackName));
+    await this.aws.deleteStacks(...stacksToDelete.map(s => {
+      if (!s.StackName) {
+        throw new Error('Stack name is required to delete a stack.');
```

Comment:
should we be throwing an error here? I don't see that we did this before... I'm wondering if throwing here would cause resources to leak in the case that this error is thrown?